src/frontends/pytorch/src/op/logaddexp.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/exp.hpp"
+#include "openvino/op/log.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_logaddexp(const NodeContext& context) {
+    // "aten::logaddexp(Tensor self, Tensor other) -> Tensor"
+    num_inputs_check(context, 2, 2);
+
+    auto input1 = context.get_input(0);
+    auto input2 = context.get_input(1);
+
+    // Convert inputs to floating point type if needed
+    input1 = context.mark_node(std::make_shared<v0::Convert>(input1, element::f32));
+    input2 = context.mark_node(std::make_shared<v0::Convert>(input2, element::f32));
```

Comment:
```suggestion
    Output<Node> input1;
    Output<Node> input2;
    std::tie(input1, input2) = get_inputs_with_promoted_types(context, 0, 1);
```
This will promote inputs to the common type, since `torch.logaddexp` require one of the inputs to be of floating type, the common type will be floating type.